library/src/fy_library/flows/create_template_model_using_parsed_fy_py_file_fy.py

```diff
@@ -0,0 +1,85 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+"""fy
+from fy_library.domain.fy_py_template_models import BaseTemplateModel
+
+
+flow CreateTemplateModelUsingParsedFyPyFile -> BaseTemplateModel:
+    property parsed_fy_py_file using setter
+    property parsed_fy_py_files_map_by_key using setter
+    property property_mixins using template_model_properties
+    property property_setter_mixins using property_mixins
+    property parsed_fy_py_files using property_setter_mixins__mapped_to_abstract_property
+    property template_models using parsed_fy_py_files__with_property_setters
+    property optional_flow_template_model_with_property_setters using parsed_fy_py_file_and_property_setters_template_models
+    property optional_base_flow_template_model_with_property_setters using parsed_fy_py_file_and_property_setters_template_models
+"""
+
+from typing import Any
+from typing import Dict
+
+from fy_core.base.flow_base import FlowBase
+
+from fy_library.domain.fy_py_template_models import (
+    BaseTemplateModel,
+)
+from fy_library.domain.parsed_fy_py_file import ParsedFyPyFile
+from fy_library.mixins.property.optional_base_flow_template_model_with_property_setters.using_parsed_fy_py_file_and_property_setters_template_models_fy import (
+    OptionalBaseFlowTemplateModelWithPropertySetters_UsingParsedFyPyFileAndPropertySettersTemplateModels_PropertyMixin,
+)
+from fy_library.mixins.property.optional_flow_template_model_with_property_setters.using_parsed_fy_py_file_and_property_setters_template_models_fy import (
+    OptionalFlowTemplateModelWithPropertySetters_UsingParsedFyPyFileAndPropertySettersTemplateModels_PropertyMixin,
+)
+from fy_library.mixins.property.parsed_fy_py_file.using_setter import (
+    ParsedFyPyFile_UsingSetter_PropertyMixin,
+)
+from fy_library.mixins.property.parsed_fy_py_files.using_property_setter_mixins__mapped_to_abstract_property__fy import (
+    ParsedFyPyFiles_UsingPropertySetterMixins_MappedToAbstractProperty_PropertyMixin,
+)
+from fy_library.mixins.property.parsed_fy_py_files_map_by_key.using_setter import (
+    ParsedFyPyFilesMapByKey_UsingSetter_PropertyMixin,
+)
+from fy_library.mixins.property.property_mixins.using_template_model_properties_fy import (
+    PropertyMixins_UsingTemplateModelProperties_PropertyMixin,
+)
+from fy_library.mixins.property.property_setter_mixins.using_property_mixin_fy import (
+    PropertySetterMixins_UsingPropertyMixins_PropertyMixin,
+)
+from fy_library.mixins.property.template_models.using_parsed_fy_py_files__with_property_setters__fy import (
+    TemplateModels_UsingParsedFyPyFiles_WithPropertySetters_PropertyMixin,
+)
+
+
+# fy:start ===>>>
+class CreateTemplateModelUsingParsedFyPyFile_Flow(
```

Comment:
That one method was worthy of a new flow - the context did not match and it could not just be expanded with a a couple of properties.